spec/default_spec.rb

```diff
@@ -9,6 +9,43 @@
       it 'converges successfully' do
         expect { chef_run }.to_not raise_error
       end
+      it 'sets node attributes' do
+        { chef_run.node['osl-varnish']['ncsa_directives'] => { fqdn: 'test.osuosl.org' } }
+      end
```

Comment:
You need to set the node attribute in the SoloRunner block and then it should work in the render_file test below. You don't make an it do block for setting node attributes. Take a look at some of the cookbooks where we use node.set['foo'].